proto/mls/api/v1/mls.proto

```diff
@@ -190,3 +222,51 @@ message GetIdentityUpdatesResponse {
   // of the request
   repeated WalletUpdates updates = 1;
 }
+
+// Sort direction for queries
+enum SortDirection {
+  SORT_DIRECTION_UNSPECIFIED = 0;
+  SORT_DIRECTION_ASCENDING = 1;
+  SORT_DIRECTION_DESCENDING = 2;
+}
+
+// Pagination config for queries
+message PagingInfo {
+  SortDirection direction = 1;
+  uint32 limit = 2;
+  uint64 cursor = 3;
```

Comment:
Does the cursor point to `id` or `created_ns`? Would it make sense to embed that into the name, e.g. `id_cursor`?

Reply:
Using the `id` - will rename to `id_cursor` :+1: